react/features/mobile/call-integration/middleware.js

```diff
@@ -139,9 +140,13 @@ function _appWillMount({ dispatch, getState }, next, action) {
  * @private
  * @returns {*} The value returned by {@code next(action)}.
  */
-function _conferenceFailed(store, next, action) {
+function _conferenceFailed({ getState }, next, action) {
     const result = next(action);
 
+    if (!isCallIntegrationEnabled(getState)) {
```

Comment:
I'm wondering if it would make sense to have this check in one place. For example by moving all the affected switch cases into a separate switch below the other initialization checks which are not conditional?